.github/workflows/ci.yaml

```diff
@@ -28,6 +28,9 @@ jobs:
 
   integration-test-lxd-charm:
     name: Integration tests for the charm (lxd)
+    needs:
+      - lint
```

Comment:
It could be worth setting this dependency on the `unit-test` job, and only depending on the `unit-test` in the integration tests.